scripts/manual_operations/manual_remove.py

```diff
@@ -242,11 +213,22 @@ def user_input_check(instrument, run_numbers):
     return user_input
 
 
-def main():
+def main(instrument: str, first_run: int, last_run: int = None):
```

Comment:
This is probably my bad providing a poor description XD What i mean by adding the types is that we add the types in the docstring themselves so something like this:
`  :param instrument: (str) Instrument to run on
    :param first_run: (str) First run to be removed
    :param last_run: (str) Optional last run to be removed`

Reply:
As soon as this is changed, you can merge in 👍

Reply:
My mistake 👍